src/expandableCalendar/week.js

```diff
@@ -30,6 +28,10 @@ class Week extends Component {
     this.style = styleConstructor(props.theme);
   }
 
+  isDateNotInTheRange = (minDate, maxDate, date) => {
+    return (minDate && !dateutils.isGTE(date, minDate)) || (maxDate && !dateutils.isLTE(date, maxDate));
+  };
+
```

Comment:
This function is same as in `Calendar`. It doesn't depend on props neither state so you could extract it somewhere outside and reuse it.

Reply:
Good idea. Moved to dateutils